libraries/helper.rb

```diff
@@ -20,7 +20,11 @@ def read_config
   config_file = ENV['INSPEC_AEM_AWS_CONF'] || './conf/aem-aws.yml'
   config = YAML.load_file(config_file) if File.exist?(config_file)
   config_params = {}
-  %w[profile access_key_id secret_access_key s3_bucket].each { |field|
+
+  # a hotfix for ruby_aem_aws
+  config_params[:region] = config['aws']['region']
```

Comment:
This is what `ruby_aem_aws` expects.

Reply:
@lucjanrudziak  thanks, I already raised the PR for ruby_aem_aws to fix this. I would remove this fix in the near future.